apps/storybook-react/.storybook/main.ts

```diff
@@ -36,7 +37,7 @@ const config: StorybookConfig = {
         ...config.resolve.alias,
         '@metamask/design-tokens': path.resolve(
           __dirname,
-          '../../../node_modules/@metamask/design-tokens',
+          '../../../packages/design-tokens',
```

Comment:
Updating reference, this can possibly be removed. Will update in https://github.com/MetaMask/metamask-design-system/issues/129